charts/prometheus-elasticsearch-exporter/templates/deployment.yaml

```
@@ -110,10 +110,10 @@ spec:
                     "--collector.slm",
                     {{- end }}
                     {{- if .Values.es.data_stream }}
-                    "--es.data_stream",
+                    "--es.data-stream",
```

I noticed that this should be `--collector.data-stream`
```suggestion
                    "--collector.data-stream",
```

---

just saw this a second before you commented, thanks!